x-pack/legacy/plugins/ml/public/index.scss

```diff
@@ -8,6 +8,11 @@
 // ML has it's own variables for coloring
 @import 'variables';
 
+// Kibana management page ML section
+#kibanaManagementMLSection {
+  @import 'management/index';
```

Comment:
I wonder if we could just import the original scss files from the jobs list and wrap them under this id? Or were there changes necessary to the SCSS imported here compared to the original one?

Reply:
The original scss file under the `ml-app` id includes a lot of Eui class overrides and ends ups breaking some of the styling in the table - that's why I had to copy some of these over. Also, as the management page changes/gets augmented it would be good to keep style changes isolated.

Reply:
Thanks for clarifying! 👍